src/scripts/feature-flags.js

```diff
@@ -0,0 +1,24 @@
+import {
+  compare
+} from './util';
```

Comment:
Just FYI, #1421 removes the need for `compare` and changes how these flags are set.